bitcoin-core-dev-tech/2017-09-05-meeting-notes.es.md

```diff
@@ -0,0 +1,278 @@
+---
+title: Notas de la reunión (2017-09-05)
+transcript_by: Bryan Bishop
+translation_by: Blue Moon
+categories: ['core-dev-tech']
+tags: ['privacidad']
+---
+
+2017-09-05 - 2017-09-07
+
+coredev.tech septiembre 2017
+
+<https://twitter.com/kanzure/status/907233490919464960>
+
+((Como siempre, cualquier error es probablemente mío, etc.))
+
+<div id="intro" />
```

Comment:
This `<div/>` is not needed

Reply:
I have removed the **HTML** elements, thank you.